browser/model/helpers/installer.js

```diff
@@ -100,15 +102,25 @@ class Installer {
         });
       } else if(zipFile.endsWith('.zip')) {
         Logger.info(this.key + ' - Extract ' + zipFile + ' to ' + extractTo);
-        fs.createReadStream(zipFile)
-          .pipe(unzip.Extract({path: extractTo}))
-          .on('close', () => {
-            Logger.info(this.key + ' - Extract ' + zipFile + ' to ' + extractTo + ' SUCCESS');
-            resolve(true);
-          })
-          .on('error', (err) => {
-            Logger.error(this.key + ' - ' + err);
-            reject(err);
+        fs.createReadStream(zipFile).pipe(unzip.Parse())
```

Comment:
What is wrong with original code?

Reply:
At first I thought you could just use the original unzip and then rename the parent folder.
But since this method is no longer used and the new code is clearly the same as https://github.com/redhat-developer-tooling/developer-platform-install/blob/master/browser/model/jbossfusekaraf.js#L27 there might be some use in having it in a lib like this.

Reply:
Ups, sorry for my confusing comment. I really meant why not to create generic installer from zip files that would unpack archive and then add some specific locations from zip to PATH. We can list files to add to PATH as part of json data.
But I am fine with current solution as well.